### PR TITLE
More Modules option removed. Placeholder text added

### LINKED ISSUE #992: Unordered list of modules
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/image-sequencer/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

**To Do**: There is an open pr #710 which adds a helper method called `scopeQuery` to image-sequencer demo code. You have to document it in the CONTRIBUTING.md file.

### 📋 Step by Step
- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [CONTRIBUTING.md](https://github.com/publiclab/image-sequencer/blob/main/CONTRIBUTING.md) in the `image-sequencer` repository (press the little pen Icon) and edit the line as shown below. 

Have a look here:
fto
The "More Modules" option has to be removed and rather can be a made a value for select.
The files to be looked into are:
https://github.com/publiclab/image-sequencer/blob/main/examples/demo.js
https://github.com/publiclab/image-sequencer/blob/main/examples/lib/intermediateHtmlStepUi.js
 And something to help out is :
https://stackoverflow.com/questions/21962124/how-to-set-a-value-for-a-selectize-js-input

- [ ] 💾 **Commit** your changes
- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review
### Please keep us updated
💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?
🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by teisenhower**
@HarshKhandeparkar 
Fixes #992

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
Hi! You can remove the `[ ]` in the issue number. Doing this will change it to a link to the issue.

**Comment by teisenhower** [comment]
@HarshKhandeparkar  Removed. Sorry about that

**Comment by HarshKhandeparkar** [comment]
No need for an apology

**Comment by teisenhower** [comment]
I do not, never done it. But I can figure out, what needs to be done?

**Comment by HarshKhandeparkar** [comment]
I'll give a link. Check [this](
https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line)
out.

**Comment by teisenhower** [comment]
I updated the file. Let me know if that resolved the issue

Thanks!

**Comment by teisenhower** [comment]
@HarshKhandeparkar @Divy123 Just want to make sure everything is ok with my PR. Is anything more I need to do to finalize this?

Thanks so much

File: examples/lib/intermediateHtmlStepUi.js [hunk 1]
```diff
@@ -118,9 +118,9 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
       sortField: 'text'
     });
     $(step.ui.querySelector('.inserDiv .add-step-btn')).prop('disabled', true);
-
-    insertStepSelect.append('<option value="none" disabled selected>More modules...</option>');
-    $(step.ui.querySelector('.insertDiv .radio-group .radio')).on("click", function () {
+    
+    insertStepSelect.append('<option value="" disabled selected>Select a Module</option>');
+    $('#insertStep .radio-group .radio').on("click", function () {
```
Comment (HarshKhandeparkar) [inline examples/lib/intermediateHtmlStepUi.js]:
```suggestion
    $(step.ui.querySelector('.insertDiv .radio-group .radio')).on("click", function () {
```

You can merge these suggested right away in the broswer and your PR will be good to go.

**Comment by teisenhower** [comment]
@HarshKhandeparkar Made your requested change. Thank you!

**Comment by teisenhower** [comment]
Thank you! Sorry for any and all confusion during this. Still getting used to the GitHub workflow.

I really appreciate your help. I hope to be able to help on this project again!

**Comment by HarshKhandeparkar** [comment]
We also hope to have your contributions to this project. :rocket:

**Comment by jywarren** [comment]
Oh, this is beautiful!!! Thanks and thanks to everyone who helped out!

**Comment by teisenhower** [comment]
@jywarren Thank you! Happy to have helped! Looking forward to the next one.

**Comment by teisenhower** [comment]
@welcome my Twitter handle is @teisenhower

Thanks!

**Pull request merged**